marketplace/music-visualizer/adjuster_scripts/effectTrailerRight.js

```diff
@@ -0,0 +1,55 @@
+//  
+//  effectTrailerRight.js
```

Comment:
Do we need to duplicate these files, or can we have a single file that works for both left/right and switches based on which trigger is being pulled?

Reply:
I will look into this